utils/ansible/ftest/vars/Rocky8.yml

```diff
@@ -6,39 +6,51 @@ daos_openmpi_module_path: /usr/share/modulefiles/mpi/openmpi-x86_64
 daos_base_deps:
   rpms:
     - bash
+    - capstone
     - gdb
     - glibc
     - jq
     - lbzip2
-    - mercury
+    - numactl
+    - numactl-libs
+    - numatop
     - platform-python
     - python36
     - systemd
     - tree
 
 daos_dev_deps:
   rpms:
+    - argobots-devel
+    - capstone-devel
+    - bzip2-devel
+    - clustershell
     - cmake
     - createrepo
+    - daos-raft-devel
     - dnf-plugins-core
+    - fuse3-devel
+    - gcc-c++
+    - gcc-gfortran
+    - golang
     - libaec-devel
+    - libattr-devel
+    - libcmocka-devel
+    - libevent-devel
+    - libyaml-devel
     - meson
+    - ninja-build
```

Comment:
clustershell, meson and ninja-build also might not be needed here.

Reply:
- [ ] Remove useless dependencies

Reply:
> * [x]  Remove useless dependencies

Fixed with commit 03ae13204aacf1821a8cd75734207322c7ed3881